django-stubs/dispatch/dispatcher.pyi

```diff
@@ -20,5 +20,6 @@ class Signal:
     def has_listeners(self, sender: Any = ...) -> bool: ...
     def send(self, sender: Any, **named: Any) -> list[tuple[Callable, str | None]]: ...
     def send_robust(self, sender: Any, **named: Any) -> list[tuple[Callable, Exception | str]]: ...
+    def _live_receivers(self, sender: Any) -> list[Callable]: ...
```

Comment:
Would be great to typehint the arguments of `Callable[]` here. But offtopic for this PR.